dcos/http.py

```diff
@@ -162,11 +174,23 @@ def request(method,
     if 'headers' not in kwargs:
         kwargs['headers'] = {'Accept': 'application/json'}
 
-    response = _request(method, url, is_success, timeout, **kwargs)
+    if verify is None and os.environ.get(constants.DCOS_SSL_VERIFY_ENV):
+        verify = os.environ.get(constants.DCOS_SSL_VERIFY_ENV)
+        if verify.lower() == "true":
+            verify = True
+        elif verify.lower() == "false":
+            verify = False
+
+    # Silence 'Unverified HTTPS request' and 'SecurityWarning' for bad certs
+    if verify is not None:
+        silence_requests_warnings()
```

Comment:
I think we do this in a few places. Should we remove them now that we do them here?